xbmc/interfaces/legacy/ModuleXbmcplugin.h

```diff
@@ -103,10 +103,59 @@ namespace XBMCAddon
      *                           - SORT_METHOD_LABEL_IGNORE_THE, SORT_METHOD_VIDEO_SORT_TITLE,
      *                           - SORT_METHOD_VIDEO_SORT_TITLE_IGNORE_THE, SORT_METHOD_FULLPATH,
      *                           - SORT_METHOD_LABEL_IGNORE_FOLDERS, SORT_METHOD_CHANNEL
+
+     *   - sortMethods available:
+     *        - xbmcplugin.SORTMETHOD_ALBUM
+     *        - xbmcplugin.SORTMETHOD_ALBUM_IGNORE_THE
+     *        - xbmcplugin.SORTMETHOD_ARTIST
+     *        - xbmcplugin.SORTMETHOD_ARTIST_AND_YEAR
+     *        - xbmcplugin.SORTMETHOD_ARTIST_IGNORE_THE
+     *        - xbmcplugin.SORTMETHOD_BITRATE
+     *        - xbmcplugin.SORTMETHOD_CHANNEL
+     *        - xbmcplugin.SORTMETHOD_CHANNEL_NUMBER
+     *        - xbmcplugin.SORTMETHOD_COUNTRY
+     *        - xbmcplugin.SORTMETHOD_DATE
+     *        - xbmcplugin.SORTMETHOD_DATEADDED
+     *        - xbmcplugin.SORTMETHOD_DATE_TAKEN
+     *        - xbmcplugin.SORTMETHOD_DRIVE_TYPE
+     *        - xbmcplugin.SORTMETHOD_DURATION
+     *        - xbmcplugin.SORTMETHOD_EPISODE
+     *        - xbmcplugin.SORTMETHOD_FILE
+     *        - xbmcplugin.SORTMETHOD_FULLPATH
+     *        - xbmcplugin.SORTMETHOD_GENRE
+     *        - xbmcplugin.SORTMETHOD_LABEL
+     *        - xbmcplugin.SORTMETHOD_LABEL_IGNORE_FOLDERS
+     *        - xbmcplugin.SORTMETHOD_LABEL_IGNORE_THE
+     *        - xbmcplugin.SORTMETHOD_LASTPLAYED
+     *        - xbmcplugin.SORTMETHOD_LISTENERS
+     *        - xbmcplugin.SORTMETHOD_MAX
+     *        - xbmcplugin.SORTMETHOD_MPAA_RATING
+     *        - xbmcplugin.SORTMETHOD_NONE
+     *        - xbmcplugin.SORTMETHOD_PLAYCOUNT
+     *        - xbmcplugin.SORTMETHOD_PLAYLIST_ORDER
+     *        - xbmcplugin.SORTMETHOD_PRODUCTIONCODE
+     *        - xbmcplugin.SORTMETHOD_PROGRAM_COUNT
+     *        - xbmcplugin.SORTMETHOD_SIZE
+     *        - xbmcplugin.SORTMETHOD_SONG_RATING
+     *        - xbmcplugin.SORTMETHOD_SONG_USER_RATING
+     *        - xbmcplugin.SORTMETHOD_STUDIO
+     *        - xbmcplugin.SORTMETHOD_STUDIO_IGNORE_THE
+     *        - xbmcplugin.SORTMETHOD_TITLE
+     *        - xbmcplugin.SORTMETHOD_TITLE_IGNORE_THE
+     *        - xbmcplugin.SORTMETHOD_TRACKNUM
+     *        - xbmcplugin.SORTMETHOD_UNSORTED
+     *        - xbmcplugin.SORTMETHOD_VIDEO_RATING
+     *        - xbmcplugin.SORTMETHOD_VIDEO_RUNTIME
+     *        - xbmcplugin.SORTMETHOD_VIDEO_SORT_TITLE
+     *        - xbmcplugin.SORTMETHOD_VIDEO_SORT_TITLE_IGNORE_THE
+     *        - xbmcplugin.SORTMETHOD_VIDEO_TITLE
+     *        - xbmcplugin.SORTMETHOD_VIDEO_USER_RATING
+     *        - xbmcplugin.SORTMETHOD_YEAR
```

Comment:
didn't you already do (340856371a0f6752cd5bc04753ec2c5510d03980) and undo (336792b266bde53c59d2ba4d5fca79b0985a799c) this once?:)